servers/src/main/java/tachyon/worker/block/evictor/GreedyEvictor.java

```diff
@@ -40,6 +40,8 @@
 
   public GreedyEvictor() {}
 
+  public GreedyEvictor(BlockMetadataManagerView view) {}
```

Comment:
Please leave a simple comment saying this evictor strategy doesn't need the view.